src/styles/NavHeader.module.less

```diff
@@ -26,6 +26,13 @@
   > li {
     margin: 0 0 0 2rem;
     padding: 0;
+    @media only screen and (max-width: 1000px) {
+      margin: 0 0 0 1rem;
+      
+    }
+    @media only screen and (max-width: 900px) {
+      margin: 0 0 0 0.5rem;
+    }
   }
```

Comment:
@toucham What are your thoughts on this fix?

Reply:
lgtm; however, `1024px` looks a bit wonky
<img width="1440" alt="Screen Shot 2022-06-16 at 22 50 59" src="https://user-images.githubusercontent.com/52475722/174215239-a88514b4-1f3a-4f35-802c-08b0aab36c8f.png">